docs/artifact-types/deb.md

```diff
@@ -0,0 +1,6 @@
+# Debian artifact (.deb)
+
+```
+$ go run ./cmd deb --package file://$PWD/dist/grafana-enterprise-10.1.0-pre_lUJuyyVXnECr_linux_amd64.tar.gz
```

Comment:
These don't need to be absolute paths technically but it still works :)

Reply:
True, but IMO it is more confusing if file:// all of a sudden is relative, hence the example with the full path :)